### PR TITLE
ss/DCOS-51357 Adding new script to RexRay doc.

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description
https://jira.mesosphere.com/browse/DCOS-51357

Document rexray nvme support prerequisites

## Urgency
- [x] Blocker 
- [ ] High
- [ ] Medium

The EBS section is growing too large to be a bullet point, so it is now a subsection.

I added the script from https://jira.mesosphere.com/browse/DCOS-51356 and broke it up into steps. 

Since there is some uncertainty in the JIRA tickets about whether this change will apply to 1.12 docs and earlier, right now the only version I have edited is 1.13.

### CODE REVIEW

File: pages/1.13/storage/external-storage/index.md [hunk 1]
```diff
@@ -133,11 +133,53 @@ If you scale your app down to 0 instances, the volume is detached from the agent
...
+
+##   EBS specific
+Volumes created on the same AWS account share a namespace. Choose unique volume names to avoid conflicts when multiple clusters are launched under the same account.
+
+EBS volumes are also namespaced by their availability zone (AZ), and an EBS volume [can only be attached to an EC2 instance in the same AZ][12]. As a result, attempts to launch a task in an agent running in a different AZ will lead to the creation of a new volume of the same name. If you create a cluster in one AZ, destroy it, be sure to create your cluster in the same AZ if you wish to reuse any external volumes. If a cluster spans multiple AZs, use Marathon constraints to only launch an instance in the same AZ.
+
+
+REX-Ray by default will fail after 13 EBS volumes are attached. While REX-Ray [0.11.0 introduced the config option `useLargeDeviceRange` to extend this limit][13], DC/OS v1.12.0 bundles REX-Ray 0.9.0.
+
+
```
Comment (gpaul) [inline pages/1.13/storage/external-storage/index.md]: s/EBS volumes are .../EBS volumes present as non-volatile memory express (NVMe) devices on certain newer EC2 instance types./

s/NVME/NVMe/

s/newer CoreOS releases/newer CoreOS AMI's/ ?

Comment (SarahStegall) [reply pages/1.13/storage/external-storage/index.md]: Okay, I've made substitutions/corrections as noted, but I can't answer your query about whether the steps noted are the default on CoreOS. See if this revision says what you want.

File: pages/1.13/storage/external-storage/index.md [hunk 2]
```diff
@@ -133,11 +133,53 @@ If you scale your app down to 0 instances, the volume is detached from the agent
...
+    
+    ```bash
+    $ yum install -y nvme-cli
+    ```
+
+1. Install the necessary udev rule and helper script. These are taken from the [RexRay user guide](https://github.com/rexray/rexray/blob/362035816046e87f7bc5a6ca745760d09a69a40c/.docs/user-guide/storage-providers/aws.md#nvme-support).
+
+    ```bash
+    $ cat <<EOF > /etc/udev/rules/999-aws-ebs-nvme.rules
+    ```
```
Comment (gpaul) [inline pages/1.13/storage/external-storage/index.md]: Please drop lines 159-161. The script starts at "cat EOF" and ends at the EOF on line 163.

File: pages/1.13/storage/external-storage/index.md [hunk 3]
```diff
@@ -133,11 +133,53 @@ If you scale your app down to 0 instances, the volume is detached from the agent
...
+1. Set EBS NVME devices.
+    ```bash
+    KERNEL=="nvme[0-9]*n[0-9]*", ENV{DEVTYPE}=="disk", ATTRS{model}=="Amazon Elastic Block Store", PROGRAM="/usr/local/bin/ebs-nvme-mapping /dev/%k", SYMLINK+="%c"
+    EOF
+    $ cat <<EOF > /usr/local/bin/ebs-nvme-mapping
+    ```
```
Comment (gpaul) [inline pages/1.13/storage/external-storage/index.md]: Please drop lines 165-167. The two EOF's demarcate a file so lines 164 and 178 form a pair and should run unbroken

File: pages/1.13/storage/external-storage/index.md [hunk 4]
```diff
@@ -133,11 +133,53 @@ If you scale your app down to 0 instances, the volume is detached from the agent
...
+    [ -n "${vol}" ] && echo "${vol/xvd/sd} ${vol/sd/xvd}"
+    EOF
+    $ chown root:root /usr/local/bin/ebs-nvme-mapping
+    $ chmod 700 /usr/local/bin/ebs-nvme-mapping
+    $ udevadm control --reload
+    ```
+
+## External volumes   
+
+To troubleshoot external volumes, consult the agent or system logs. If you are using REX-Ray on DC/OS, you can also consult the `systemd` journal.
```
Comment (gpaul) [inline pages/1.13/storage/external-storage/index.md]: "...consult the `systemd` journal for the `dcos-rexray.service` unit logs.

File: pages/1.13/storage/external-storage/index.md (hunk 3 above)
Comment (SarahStegall) [reply pages/1.13/storage/external-storage/index.md]: Done!

File: pages/1.13/storage/external-storage/index.md (hunk 4 above)
Comment (SarahStegall) [reply pages/1.13/storage/external-storage/index.md]: Done!

File: pages/1.13/storage/external-storage/index.md (hunk 2 above)
Comment (SarahStegall) [reply pages/1.13/storage/external-storage/index.md]: Done!

**Pull request merged**